### ISSUE TITLE
The login and sign up page  inputBox is lacking focus styles

### ISSUE THREAD

**Issue by SahasSaurav** [divanov11/Mumble#359]
The login and signup pages  form element is lacking focus styles and which is not great for accessibility

**Comment by MidouWebDev** [comment]
@SahasSaurav can you provide screenshots ?

**Comment by SahasSaurav** [comment]
![Screenshot (151)](https://user-images.githubusercontent.com/60291223/118112959-50e4b700-b403-11eb-8bb1-748105f51b34.png)
![Screenshot (150)](https://user-images.githubusercontent.com/60291223/118112983-56da9800-b403-11eb-88cf-c0907f282d13.png)

**Comment by MidouWebDev** [comment]
@SahasSaurav 

Do you want to work on it ?
Do you have a preview of what you expected to have on the login/register page

**Comment by SahasSaurav** [comment]
Yes I can work on this actually it

**Comment by MidouWebDev** [comment]
@SahasSaurav okey cool
I will assign you

**Comment by SahasSaurav** [comment]
Hey where i can interact with the designer  so we can discuss color for focus state and  how to want to show focus input box
This is how i have done my project

![Screenshot (152)](https://user-images.githubusercontent.com/60291223/118150251-f7df4800-b42f-11eb-9f55-f284aaf84028.png)

**Comment by MidouWebDev** [comment]
@SahasSaurav Did you join the Mumble discord server ?

**Comment by codyseibert** [comment]
Note: the light mode adds and outline to the focused input box, so maybe this is an issue for dark mode only?

Also, we need to add autofocus the first input on page load so the user can start typing when the page loads.

**Comment by MidouWebDev** [comment]
@SahasSaurav

**Comment by MidouWebDev** [comment]
@SahasSaurav are you working on it ?

**Comment by okumujustine** [comment]
> @SahasSaurav are you working on it ?

I can take up this task in case @SahasSaurav  is busy, and if he is okay with me taking over.

**Comment by okumujustine** [comment]
I created a PR for this issue, however, I wasn't assigned hopefully we can go with it if it wasn't solved yet.

**Comment by MidouWebDev** [comment]
Thanks @okumujustine 
I reviewed your PR
Everything is good ! 👍🏻

**Comment by MidouWebDev** [comment]
@SahasSaurav i am so sorry
But there is other issues in the repo if you want to contribute to it
Sorry

**Comment by MidouWebDev** [comment]
#424
